src/content/docs/infrastructure/prometheus-integrations/integrations-list/redpanda-integration.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Redpanda integration
+tags:
+    - Integrations
+    - Prometheus integrations
+    - Configure Prometheus OpenMetrics for Redpanda
+metaDescription: Get dashboards of all your most important Redpanda metrics with our Redpanda quickstart. 
+---
+
+import prometheusRedpandaIntegration from 'images/prometheus_screenshot-full_redpanda-integration.png'
+
+With our Redpanda dashboards, you can track your cluster-level metrics, Redpanda scheduling groups, and your service errors and uptime. Built with our infrastructure agent and our Prometheus OpenMetrics integration, our Redpanda dashboard take advantage of OpenMetrics endpoint scraping, so you can view all your most important data, all in one place.
```

Comment:
I'd add the URLs for the infrastructure agent and for the Prometheus OpenMetrics integration here